### ISSUE TITLE
[Adaptive Card iOS - DeepFallBack.Json]: Voice over does not announce the 'Dimmed' information for the dimmed text.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7355]
https://microsoft.visualstudio.com/OS/_workitems/edit/37844858
